tunix/generate/vllm_sampler.py

```diff
@@ -164,6 +178,12 @@ def tokenize(self, input_string: str) -> List[int]:
         if input_ids[-1] != self.tokenizer.eos_id()
         else []
     )
+    # return bos_tok + input_ids + eos_tok
+    eos_tok = (
```

Comment:
are you repeating above logic? I think line 181 can be uncommented and then then you don't need to add new logic here?

Reply:
ah yes - sorry - done!